### PR TITLE
docs: updates ranges, adds docs on environment URL [sc-0000]

### PR DESCRIPTION
**Pull request by tnolet**

### CODE REVIEW

File: site/content/docs/CICD/vercel.md [hunk 1]
```diff
@@ -55,6 +55,47 @@ Once the project and the check/group have been linked, you are able to specify w
...
+
+
+```js {hl_lines=[7]}
+const { chromium } = require('playwright')
+
+(async () => {
+  const browser = await chromium.launch()
+  const page = await browser.newPage()
+
+  const targetUrl = process.env.ENVIRONMENT_URL || 'https://www.checklyhq.com'
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:76]: I'd rather the example URL be anything but checklyhq.com for more immediate understanding. Maybe example.com? https://demo.vercel.store?

File: site/content/docs/CICD/vercel.md [hunk 2]
```diff
@@ -99,10 +140,10 @@ These ranges are listed in the table below and are based on the guidelines devel
 
 | KPI | good       | needs improvement  | poor     |
 |-----|------------|--------------------|----------|
-| FCP | <= 1.8 sec | > 1.8 sec <= 3 sec | > 3 sec  |
-| LCP | <= 2.5 sec | > 2.5 sec <= 4 sec | > 4 sec  |
+| FCP | <= .93 sec | > .93 sec <= 1.6 sec | > 1.6 sec  |
```
Comment (ragog) [inline site/content/docs/CICD/vercel.md:143]: Can't comment on these so I will assume they are correct.

**Review by ragog — APPROVED** [review]
Just a nit @tnolet. I leave it up to you if that needs changing or not. Otherwise LGTM.

File: site/content/docs/CICD/vercel.md (hunk 2 above)
Comment (tnolet) [reply site/content/docs/CICD/vercel.md:143]: @ragog yeah, this is straight from the source code.

File: site/content/docs/CICD/vercel.md (hunk 1 above)
Comment (tnolet) [reply site/content/docs/CICD/vercel.md:76]: @ragog, maybe I should add our own website runs on Vercel? I kinda like showing exactly what we use ourselves i.e. dogfooding.

Comment (ragog) [reply site/content/docs/CICD/vercel.md:76]: @tnolet I see why that would be valuable. I am just trying to think about this from the perspective of a user who might be new to the integration, Vercel, Checkly, Playwright (possibly all the same time). Seeing the Checkly URL there might create confusion, e.g. "so I need Playwright to talk to Checkly, and this is how I do it." But the target needs to clearly be the user's own site. I try to be careful about these things when writing tuts, leaving the least amount of room for possible misunderstandings. A way to fight the Curse of Knowledge if you will.

**Review by ragog — APPROVED** [review]
Re-approved, see suggestion in comment.

**Pull request merged**